Automattic/wp-calypso
# Stats: Swapping `File downloads` component

#### Proposed Changes

* using a feature flag and replacing `File downloads` StatsModule with the recently created component with horizontal bars
* extending component used in #69545 and #69726 with 
    * a translation for component's numeric column,
    * extending action generating component to accept `children` in case `action` data is not available (e.g. File downloads)
    
This PR builds on #69726 so should be merged after it.

#### Testing Instructions



* Navigate to the main Stats page either on a local instance or the test URL from the comment below
* append `?flags=stats/new-stats-module-component` to the URL
* verify that `File downloads` component has horizontal bars

Before:
![Screenshot 2022-11-07 at 4 19 26 PM](https://user-images.githubusercontent.com/112354940/200219520-709f3c01-424e-40ec-b02b-1a573ff55d9d.png)

After:
![Screenshot 2022-11-07 at 4 14 29 PM](https://user-images.githubusercontent.com/112354940/200219542-c158d6bc-b48a-443d-92fc-4b5d780646be.png)


#### Pre-merge Checklist






Related to #69044
